src/components/Charts/Charts.style.ts

```diff
@@ -105,20 +103,23 @@ export const TextAnnotation = styled.g`
     font-family: ${charts.fontFamily};
     font-weight: ${charts.fontWeight};
     font-size: ${charts.fontSize};
-    fill: ${palette.chart.annotation};
+    fill: ${props => palette(props).annotation};
     text-anchor: ${props => props.textAnchor || 'middle'};
     dominant-baseline: ${props => props.dominantBaseline || 'middle'};
   }
 `;
 
 export const RegionAnnotation = styled(TextAnnotation)<{ isActive: boolean }>`
   rect {
-    fill: ${props => (props.isActive ? props.color : palette.white)};
-    stroke: ${props => (props.isActive ? props.color : palette.lightGray)};
+    fill: ${props =>
+      props.isActive ? props.color : palette(props).background};
+    stroke: ${props =>
+      palette(props).regionAnnotationStroke(props.isActive, props.color)};
```

Comment:
Is there a way to know which theme is active? If not, I think that the function is good enough 

```js
stroke: ${props => (props.isActive || theme.isDarkMode) ? props.color : lightGrey}
```

Reply:
Done, thanks!